### DISCUSSION TITLE
automatic creation of an info@ alias in case of message authentication failure/improper forwarding?

### DISCUSSION THREAD

**Discussion by julmoser** [anonaddy/anonaddy#594] [Q&A] [answered]
Last week I've received an email (that is clearly spam) to an alias of a custom domain that I hadn't created myself: "info@customdomain" 
The message itself is labeled with:
"Warning: This email may be spoofed or improperly forwarded, please check the 'X-AnonAddy-Authentication-Results' header."

In the anonaddy dashboard, the alias shows as creation date the date I've received the email - but I surely hadn't created it.

Is this something anonaddy does automatically in case of authentication failure/improper forwarding? Or do I need to worry that my account has been hacked? 
Thanks

**Comment by willbrowningme** [comment, marked as answer]
Hi there,

This is simply because you have catch-all enabled for your custom domain (it's enabled by default when you add a custom domain to addy.io). This means that any alias will be created automatically (on-the-fly) as soon as it receives its first email.

So some spammer or bot just sent an email to a commonly used local part (info) for your domain and it was created automatically. Your account has not been hacked.

You can turn off catch-all for your domain from the domains page (https://app.addy.io/domains) by turning the toggle switch to OFF. Then your domain will only forward messages for aliases that already exist.

**Reply by julmoser** [reply]
Right, makes sense. Thanks a ton
